cnxepub/formatters.py

```diff
@@ -347,8 +351,8 @@ def _replace_tex_math(node, mml_url, depth=0):
         else:
             logger.warning('Retrying math TeX conversion: '
                            '{}'.format(json.dumps(eq, indent=4)))
-            if depth < 2:
-                return _replace_tex_math(node, mml_url, depth)
+            if retry < 2:
+                return _replace_tex_math(node, mml_url, retry)
```

Comment:
Perhaps a loop should be used rather than recursion. This puts an unnecessary parameter on the function.